getnotes/getnotes.py

```diff
@@ -14,6 +14,9 @@
 from redbot.core.utils.chat_formatting import pagify, box
 from redbot.core.utils.menus import menu, DEFAULT_CONTROLS
 
+#Util Imports
+from util import key_to_ckey
```

Comment:
Your utility isn't in the right place so you can't even import this. Move it into the getnotes directory and then correct your import

```suggestion
from .util import key_to_ckey
```